README.md

```diff
@@ -408,7 +408,9 @@ Do not use version 1.0.0 of Prometheus: https://groups.google.com/forum/#!topic/
 
 Even if the module has templates for several linux distributions, only RedHat family distributions were tested.
 
-This module has unit tests for CentOS/RHEL, Debian, Ubuntu and Archlinux. Acceptance tests are executed for CentOS, Debian and Ubuntu. Other operating systems may work but are untested.
+This module has unit tests for CentOS/RHEL, Debian, Ubuntu and Archlinux. Acceptance tests are executed for CentOS, Debian, Ubuntu and Archlinux.
```

Comment:
I think we should just drop those statements. We will never update them when tests are added or removed.